src/components/ProductListContainer/index.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import ProductCard from '../ProductCard'
+import './styles.css'
+import PropTypes from 'prop-types'
+
+const fakeProductList = [
+  {
+    id: 1,
+    productPhotoURL: 'https://placehold.co/308x210',
+    listPrice: 100,
+    discountedPrice: 80,
+    description: 'Este es un producto',
+    colors: 'rojo, verde, negro',
+    occupancyPercentage: 50,
+    groupCapacity: 10,
+    suscriptorsNumber: 5,
+    expirationDate: new Date().toLocaleTimeString(),
+    discountPercentage: 20
+  },
+  {
+    id: 2,
+    productPhotoURL: 'https://placehold.co/308x210',
+    listPrice: 100,
+    discountedPrice: 80,
+    description: 'Este es un producto',
+    colors: 'rojo, verde, negro',
+    occupancyPercentage: 50,
+    groupCapacity: 10,
+    suscriptorsNumber: 5,
+    expirationDate: new Date().toLocaleTimeString(),
+    discountPercentage: 20
+  },
+  {
+    id: 3,
+    productPhotoURL: 'https://placehold.co/308x210',
+    listPrice: 100,
+    discountedPrice: 80,
+    description: 'Este es un producto',
+    colors: 'rojo, verde, negro',
+    occupancyPercentage: 50,
+    groupCapacity: 10,
+    suscriptorsNumber: 5,
+    expirationDate: new Date().toLocaleTimeString(),
+    discountPercentage: 20
+  },
+  {
+    id: 4,
+    productPhotoURL: 'https://placehold.co/308x210',
+    listPrice: 100,
+    discountedPrice: 80,
+    description: 'Este es un producto',
+    colors: 'rojo, verde, negro',
+    occupancyPercentage: 50,
+    groupCapacity: 10,
+    suscriptorsNumber: 5,
+    expirationDate: new Date().toLocaleTimeString(),
+    discountPercentage: 20
+  },
+  {
+    id: 5,
+    productPhotoURL: 'https://placehold.co/308x210',
+    listPrice: 100,
+    discountedPrice: 80,
+    description: 'Deberia aparecer en la primer lista y tercera',
+    colors: 'rojo, verde, negro',
+    occupancyPercentage: 50,
+    groupCapacity: 10,
+    suscriptorsNumber: 5,
+    expirationDate: new Date().toLocaleTimeString(),
+    discountPercentage: 20
+  }
+]
+
+const getProductsFilteredById = (productIds) => {
+  return fakeProductList.filter(product => productIds.includes(product.id))
+}
+
+const ProductListContainer = ({ productIds, featuredProductId }) => {
+  return (
+    <div className='cards'>
+      {
+
+        getProductsFilteredById(productIds).map(({
```

Comment:
I really like this, because you are not using `fakeProductList` directly. The container _**doesn't know**_ that the data is fake